src/workspace-data/data-table/wds/SingleEntityEditorWds.js

```diff
@@ -0,0 +1,88 @@
+import _ from 'lodash/fp';
+import { Fragment, useState } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { ButtonPrimary, ButtonSecondary, spinnerOverlay } from 'src/components/common';
+import Modal from 'src/components/Modal';
+import { reportError } from 'src/libs/error';
+
+import { getAttributeType } from './attribute-utils';
+import AttributeInput from './AttributeInput';
+
+export const SingleEntityEditorWds = ({
+  entityType,
+  entityName,
```

Comment:
Since this component is specific to WDS, can we avoid carrying over the "entity" naming from Entity Service components?

```suggestion
  recordType,
  recordId,
```